requirements_for_test.txt

```diff
@@ -1,29 +1,32 @@
-bandit==1.6.2
+bandit~=1.7.4
 boto3
 botocore
-coveralls==2.1.2
-flake8==3.7.7
-freezegun==1.2.2
-moto==4.0.10
+coveralls~=3.3.1
+flake8==3.7.7  # TODO - upgrade to newest (6.0.0); will cause new failures
```

Comment:
This can be a new ticket.  There are a lot of new failures.

Reply:
see above re TODOs and the need for a corresponding ticket and that ticket number added to the comment.

Reply:
#1105